mdaviz/mainwindow.py

```diff
@@ -17,71 +16,62 @@
 from .opendialog import DIR_SETTINGS_KEY
 
 UI_FILE = utils.getUiFileName(__file__)
+MAX_FILES = 100
```

Comment:
-#57 

For more than NNN files, could show a dialog with a progress bar and a Cancel button.  As a UI, this leaves users the option to bail out if it takes too long.  (Add as enhancement issue).

Reply:
The `tqdm` package is great for progress bars.

Reply:
created #142